tests/test_rules.py

```diff
@@ -380,10 +380,10 @@ def test_number_symbol():
     children = list(r.statement.get_children())
     assert (Number(1) in children) == True
     assert (Number(0xFFFFFFFF) in children) == True
-    assert (Number(2, "symbol name") in children) == True
-    assert (Number(3, "symbol name") in children) == True
-    assert (Number(4, "symbol name = another name") in children) == True
-    assert (Number(0x100, "symbol name") in children) == True
+    assert (Number(2, description="symbol name") in children) == True
+    assert (Number(3, description="symbol name") in children) == True
+    assert (Number(4, description="symbol name = another name") in children) == True
+    assert (Number(0x100, description="symbol name") in children) == True
```

Comment:
@Ana06 re: our discussion on kwargs, here's an example of why I like to pass them explicitly like `kwarg=value`.

in this PR, i add the `arch=foo` kwarg to `Number`, ordering it before `description=foo` for consistency. but, this means that these tests started failing, because `"symbol name"` got passed in as the arch. by passing all kwargs by name, then updates to the method signatures (such as adding a new kwarg) don't break anything.

Reply:
This makes a lot of sense! Thanks for the explanation 😄